loader/loader_test.go

```diff
@@ -3409,3 +3409,24 @@ services:
 		"myhost": []string{"0.0.0.1", "0.0.0.2"},
 	})
 }
+
+func TestLoadDependsOnX(t *testing.T) {
+	p, err := loadYAML(`
+name: load-depends-on-x
+services:
+  test:
+    image: test
+    depends_on:
+      - x-foo
+  foo:
```

Comment:
It should be `x-foo`, no?
```suggestion
  x-foo:
```

Reply:
indeed, not actually used by the test